src/app/productDetail/templates/productDetail.html

```diff
@@ -5,12 +5,16 @@
     <hr>
     <div class="row">
         <div class="col-md-6">
-            <!--PRODUCT IMAGE-->
-            <div class="thumbnail">
+            <!-- PRODUCT IMAGES -->            
+            <div class="thumbnail" style="max-width: 400px; height: auto; margin: 0 auto;" ng-if="!productDetail.item.xp.additionalImages || !productDetail.item.xp.additionalImages.length">
```

Comment:
@amrarick26 @spencerwalker @crhistianr Do y'all think it would make sense to roll the default image display (when there are no additional images) into the oc-product-images directive?  That way we just have to move that one directive around when coming up with new PDT layouts.